### ISSUE TITLE
Make watchman settings importable

### ISSUE THREAD

**Issue by jonespm** [mwarkentin/django-watchman#130]
The django-debug-toolbar had a fix a few years ago to make it "import safe" https://github.com/jazzband/django-debug-toolbar/issues/801

This basically changed up the settings so that anything calling getenv wasn't called directly at the time. This is useful to be able to import the default attributes and configuration from a local settings

For instance this doesn't seem to be possible in (local) settings.py

collectstatic/django will either exception out about SECRET_KEY or even if this is imported after SECRET_KEY is defined then the locally defined WATCHMAN_CHECKS aren't set.

```
from watchman import settings as wm_settings

. . .

WATCHMAN_CHECKS = wm_settings.DEFAULT_CHECKS + ('dashboard.common.watchman.custom_checks.check_crontab', )
```

I think either the "constant's" could be moved to a separate constants file or the fix similar to the fix for 801 could be used. The workaround is just to redefine the the full set of DEFAULT_CHECKS locally but those "might" change in a future version and then would automatically update! 

Thanks!

**Comment by mwarkentin** [comment]
Is this similar to what was discussed in #13? I'd be open to a PR to improve this - extracting into a standalone `constants.py` seems like it would be the simplest approach. Any downsides?

**Comment by jonespm** [comment]
I created a simple PR for just moving out these 2 constant tuples to a separate file, tested it locally and updated the README.

**Comment by mirceaciu** [comment]
small impediment (maybe?). I just did a `pip install django-watchman` and this merge does not seem to be built (I don't have the separate constants file)

![image](https://user-images.githubusercontent.com/9830334/54609218-2b460800-4a5b-11e9-9e7d-e766c39409c6.png)

**Comment by jonespm** [comment]
Yeah, it looks like there hasn't been a new version released since 0.15.0 on Feb 2018...

It looked like I'd locally been installing and using this still as something like
pip install git+https://github.com/mwarkentin/django-watchman@master#egg=django-watchman

And didn't notice there wasn't a new version released yet.

**Comment by mirceaciu** [comment]
my comment seem to relate to https://github.com/mwarkentin/django-watchman/issues/136

**Comment by mwarkentin** [comment]
Oops, sorry about that. I'll try to get a new release out shortly!

**Comment by mwarkentin** [comment]
I've pushed [django-watchman 0.16.0](https://pypi.org/project/django-watchman/0.16.0/) to pypi.. hope that helps!
